e2e/test/scenarios/admin/datamodel/metadata.cy.spec.js

```diff
@@ -132,6 +133,24 @@ describe("scenarios > admin > datamodel > metadata", () => {
       .should("have.length", 1);
   });
 
+  it("should display breakouts group for all FKs (metabase#36122)", () => {
+    cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
+      semantic_type: "type/FK",
+      fk_target_field_id: PRODUCTS.ID,
+    });
+
+    openReviewsTable({ mode: "notebook" });
+    summarize({ mode: "notebook" });
+    getNotebookStep("summarize")
+      .findByText("Pick a column to group by")
+      .click();
+
+    popover().within(() => {
+      cy.findAllByText("Rating").should("have.length", 2);
+      cy.get(".List-section-header").should("have.length", 3);
```

Comment:
why so and not `findByText` with explicit values?

Reply:
Added more assertions: 16a82e26534a56ce456327e62a9f18b3c0b6130a

Reply:
I meant a more general like `cy.findByText("Review").should('exist')`, but this should work as well

Reply:
We're testing for "Rating" which should appear both as section header and as an item within one of the sections